README.md

```diff
@@ -177,6 +178,18 @@ Ensure that the motion time is configured properly, even when `position_url` is
 
 `motion_time` is the time, in milliseconds, for your blinds to move from up to down. This should only include the time the motor is running. Filming this with your phone to determine the time may be easier than trying to do it with a timer. **NOTE**: If you are performing multiple blind requests simultaneously and are getting network timeouts due to your configuration, try using non-identical `motion_time` (e.g., 9800, 10000, 10200 vs. 10000 for each) it may help.
 
+There are additional two configurations parameters, `motion_down_time` and `motion_up_time`.        
+
+`motion_down_time` is the time, in milliseconds, for your blinds to move from up to down. Everything else is exactly as described in `motion_time` above.
+
+`motion_up_time` is the time, in milliseconds, for your blinds to move from down to up. Everything else is exactly as described in `motion_time` above.
+
+The reason for this, is that there are some blinds that their opening time is different from closing time, due to simple physics - The weight of the shutter affects on the speed of the motor.  
+Ideally, a better approach would be using some kind of gradual equation for calculating the exact time, according to the position, because the weight gradually added or subtracted during the move. But, hopefully this will be a nice to have feature in the future.
+
+**Note!**  
+`motion_down_time` and `motion_up_time` are always have higher priority over `motion_time`. This means, that if all three explicitly provided in the configuration file, the value set in `motion_down_time` and `motion_up_time` will be used instead and the value in `motion_time` will be ignored. You can also provide only `motion_down_time` and `motion_up_time` value without providing `motion_time`. If only `motion_time` is provided, its value will be used for `motion_down_time` and `motion_up_time`.
+
```

Comment:
```suggestion
**Note!**  
For cases where `motion_time` varies based on the direction of shutter movement (i.e., due to gravity), `motion_down_time` and `motion_up_time` may be used for more fine-tuning.

- `motion_down_time` is the time, in milliseconds, for your blinds to move from up to down.
- `motion_up_time` is the time, in milliseconds, for your blinds to move from down to up.
- Everything else is exactly as described in `motion_time` above.

Ideally, a better approach would be using some kind of equation for calculating the exact time. This would be a nice-to-have feature in the future.

**Note!**  
`motion_down_time` and `motion_up_time` have a higher priority over `motion_time`. This means, that if all three are explicitly provided in the configuration file, the value set in `motion_time` will be ignored.
```

Reply:
Yes. Much more readable :)

Reply:
@slavikme can you commit the change? :)

Reply:
Will do